Marlin/example_configurations/BQ/Hephestos_2/Configuration.h

```diff
@@ -1088,16 +1062,11 @@
 // Add a menu item to move between bed corners for manual bed adjustment
 //#define LEVEL_BED_CORNERS
 
-#if ENABLED(LEVEL_BED_CORNERS)
-  #define LEVEL_CORNERS_INSET 30    // (mm) An inset for corner leveling
-  //#define LEVEL_CENTER_TOO        // Move to the center after the last corner
-#endif
```

Comment:
This is a regression, and not the only one. We want this feature and other updated features to be included in all configs.